Makefile

```diff
@@ -1,89 +1,45 @@
 export GO15VENDOREXPERIMENT := 1
 
-UNAME := $(shell uname)
-ifeq ($(UNAME), Darwin)
-    HASH := md5
-else
-    HASH := md5sum
-endif
-
 all: build manifests
 
-generate: sync
-	find pkg/ -name "*generated*.go" -exec rm {} -f \;
-	./hack/build-go.sh generate ${WHAT}
-	goimports -w -local kubevirt.io cmd/ pkg/ tests/
-	./hack/bootstrap-ginkgo.sh
-	(cd tools/openapispec/ && go build)
-	tools/openapispec/openapispec --dump-api-spec-path api/openapi-spec/swagger.json
+generate:
+	hack/dockerized hack/glide-checksync.sh && ./hack/generate.sh
 
-apidocs: generate
-	./hack/gen-swagger-doc/gen-swagger-docs.sh v1 html
+apidocs:
+	hack/dockerized hack/glide-checksync.sh && ./hack/generate.sh && ./hack/gen-swagger-doc/gen-swagger-docs.sh v1 html
```

Comment:
Guess you could just depend on generate target, the extra spin-up/down shouldn't take too much extra time.

Reply:
Would look a little bit nicer, but at least after you have everything in the cache it makes a noticeable difference.